docs/reference/cat/tasks.asciidoc

```diff
@@ -0,0 +1,113 @@
+[[cat-tasks]]
+=== cat task management API
+++++
+<titleabbrev>cat task management</titleabbrev>
+++++
+
+beta::[The cat task management API is new and should still be considered a beta feature. The API may change in ways that are not backwards compatible.]
+
+Returns information about tasks currently executing in the cluster,
+similar to the <<tasks,task management>> API.
+
+
+[[cat-tasks-api-request]]
+==== {api-request-title}
+
+`GET /_cat/_tasks/<task_id>`
+
+`GET /_cat/_tasks`
+
+
+[[cat-tasks-api-desc]]
+==== {api-description-title}
+
+The cat task management API returns information
+about tasks currently executing
+on one or more nodes in the cluster.
+It is a more compact view
+of the JSON <<tasks,task management>> API.
+
+
+[[cat-tasks-api-path-params]]
+==== {api-path-parms-title}
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=task-id]
```

Comment:
Looking at the code, it should also support `detailed`, `wait_for_completion` and `timeout` flags. They are not as useful in the cat context but we can add them for completeness.

Reply:
Thanks! These are included under the [query parameters section](https://github.com/elastic/elasticsearch/pull/47321/files#diff-4536eb2d8b49cc5f74c52b6d2348094cR38).